spec/unit/resource/api1600/synergy/volume_attachment_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::API1200::Synergy::VolumeAttachment do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API1200::C7000::VolumeAttachment' do
+    expect(described_class).to be < OneviewSDK::API1200::C7000::VolumeAttachment
```

Comment:
When I see lib, it seems like this class should inherit from API1600::C7000. If so, please update comment accordingly.